src/shared/components/DataComponent/DataQueryComponent.tsx

```diff
@@ -49,6 +49,15 @@ export const DataQueryComponent: FunctionComponent<DataQueryComponentProps> = ({
 					);
 				}
 
+				if (isEmpty(get(result, 'data'))) {
+					// Temp empty because of cache clean
+					return showSpinner ? (
```

Comment:
In dit geval, past een if-statement beter dan een ternary want het geeft toch geen code-length voordelen. 

Maakt het trouwens zelfs korter als je er gewoon het volgende van maakt:
```
if (isEmpty(get(result, 'data')) && showSpinner) {
	// execute
}
```

Reply:
Ja maar die code doet niet hetzelfde
als result.data niet empty is en showSpinner = true dan moet de code null returnen